src/encoded/audit/functional_characterization_experiment.py

```diff
@@ -129,7 +129,9 @@ def audit_experiment_replicate_with_no_files(value, system, excluded_statuses):
 def audit_experiment_mixed_expression_measurement_methods(value, system, excluded_types):
     if value['status'] in excluded_types:
         return
-    if value['assay_term_name'] != 'CRISPR screen':
+    if value['assay_term_name'] not in ['proliferation CRISPR screen',
```

Comment:
I don't think the proliferation CRISPR screens have any expression methods specified. I think it should be removed form the list